tests/objects/test_label_structure.py

```diff
@@ -0,0 +1,948 @@
+radio_nested_option_1 = all_types_structure.get_item_by_hash("MTExNDQ5")
+radio_nested_option_1_text = all_types_structure.get_item_by_hash("MjE2OTE0")
+radio_nested_option_2 = all_types_structure.get_item_by_hash("MTcxMjAy")
+radio_nested_option_2_checklist = all_types_structure.get_item_by_hash("ODc666Ay")
+radio_nested_option_2_checklist_option_1 = all_types_structure.get_item_by_hash("MT66665Q3")
+radio_nested_option_2_checklist_option_2 = all_types_structure.get_item_by_hash("Nzg66665E3")
+
+keypoint_dynamic = all_types_structure.get_item_by_hash("MTY2MTQx")
+dynamic_text: TextAttribute = all_types_structure.get_item_by_hash("OTkxMjU1")
+dynamic_checklist = all_types_structure.get_item_by_hash("ODcxMDAy")
+dynamic_checklist_option_1 = all_types_structure.get_item_by_hash("MTE5MjQ3")
+dynamic_checklist_option_2 = all_types_structure.get_item_by_hash("Nzg3MDE3")
+dynamic_radio = all_types_structure.get_item_by_hash("MTExM9I3")
+dynamic_radio_option_1 = all_types_structure.get_item_by_hash("MT9xNDQ5")  # This is dynamic and deeply nested.
+dynamic_radio_option_2 = all_types_structure.get_item_by_hash("9TcxMjAy")  # This is dynamic and deeply nested.
+
+text_classification = all_types_structure.get_item_by_hash("jPOcEsbw", Classification)
+text_classification_attribute: TextAttribute = all_types_structure.get_item_by_hash("OxrtEM+v", TextAttribute)
+radio_classification = all_types_structure.get_item_by_hash("NzIxNTU1")
+radio_classification_option_1 = all_types_structure.get_item_by_hash("MTcwMjM5")
+radio_classification_option_2 = all_types_structure.get_item_by_hash("MjUzMTg1")
+radio_classification_option_2_text = all_types_structure.get_item_by_hash("MTg0MjIw")
+checklist_classification = all_types_structure.get_item_by_hash("3DuQbFxo")
+checklist_classification_option_1 = all_types_structure.get_item_by_hash("fvLjF0qZ")
+checklist_classification_option_2 = all_types_structure.get_item_by_hash("a4r7nK9i")
+
+BOX_COORDINATES = BoundingBoxCoordinates(
+    height=0.1,
+    width=0.2,
+    top_left_x=0.3,
+    top_left_y=0.4,
+)
+
+POLYGON_COORDINATES = PolygonCoordinates(
+    values=[
+        PointCoordinate(x=0.2, y=0.1),
+        PointCoordinate(x=0.3, y=0.2),
+        PointCoordinate(x=0.5, y=0.3),
+        PointCoordinate(x=0.6, y=0.5),
+    ]
+)
+
+KEYPOINT_COORDINATES = PointCoordinate(x=0.2, y=0.1)
+
+
+# =======================================================
+# =========== demonstrative tests below here ============
+# =======================================================
+
+
+@pytest.mark.skip("The functionality is not fully implemented")
+def test_upload_simple_data():
+    project = "code that generates a project ..."
+    label_hash = "1234"
+
+    label_row = project.get_label_row_class(label_hash)  # can either take label_hash or data_hash
+
+    # Now create an object and add it to the label row
+    box_ontology_item = example_ontology_structure.get_item_by_hash("MjI2NzEy", Object)
```

Comment:
Should we (instead or in-addition-to) have specific `get_classification` and `get_object` instead that raise (or return `None`?) if the thing with the stated hash is not the right kind?

Reply:
so the option is here to have the `get_item_by_hash` with the member that sets the expected class, or just have a bunch of getters like `get_classification` and `get_text_attribute`, ... 
I haven't really made up my mind on what is better